src/layer/__tests__/cache.ts

```diff
@@ -113,3 +113,188 @@ describe('Testing caching', () => {
     expect(mockNetwork.history.post.length).toBe(2);
   });
 });
+
+describe('Testing cache targets', () => {
+  beforeEach(() => {
+    Object.assign(global, makeServiceWorkerEnv(), fetch);
+    jest.resetModules();
+    memoryCache.clear();
```

Comment:
For consistency, should we use `invalidateCaches` before each test as it is done in the other test files? I assume other caches should be cleared too, not just memory?

Reply:
Done